internal/cli/inputs.go

```diff
@@ -46,21 +46,17 @@ func (i *ProjectInputs) Resolve(ui terminal.UI, wd string) error {
 		return appErr
 	}
 
-	var questions []*survey.Question
 	if i.App == "" {
-		questions = append(questions, &survey.Question{
-			Name: appFieldName,
-			Prompt: &survey.Input{
-				Message: "App Filter",
-				Default: app.Option(),
-			},
-		})
-	}
-
-	if len(questions) > 0 {
-		if err := ui.Ask(i, questions...); err != nil {
-			return err
+		var appOption string
+
+		if app.RootDir != "" {
+			appOption = app.Option()
+		} else {
+			if err := ui.AskOne(&appOption, &survey.Input{Message: "App Filter"}); err != nil {
```

Comment:
this change will now only prompt the user for `App Filter` if the working directory does not have app information present.  if app details are present, then the app will be resolved accordingly (i.e. locking the app context in when running commands from a project directory)